src/components/application_manager/src/rpc_handler_impl.cc

```diff
@@ -159,10 +159,6 @@ void RPCHandlerImpl::ProcessMessageFromHMI(
   smart_objects::SmartObjectSPtr smart_object =
       std::make_shared<smart_objects::SmartObject>();
   bool allow_unknown_parameters = false;
-  if (!smart_object) {
```

Comment:
Reason for this change?

Reply:
Since `smart_object` is a shared ptr which is allocated with `std::make_shared`, i don't think that null pointer check makes sense here.